index.html

```diff
@@ -41,7 +46,169 @@ <h2 class="enphasis">I'm a software developer</h2>
+            <section class="info">
+                <h3 class="text">
+                    Multi-Post Stories Gain+Glory
+                </h3>
+                <ul class="languages">
+                    <li>HTML</li>
+                    <li>CSS</li>
+                </ul>
+                <a class="button" href="">See project</a>
+            </section>
+        </section>
+
+        <section class="card">
+            <section class="screenshot">
+                <img src="/Images/placeholder.png" alt="screenshot">
+            </section>
+            
+            <section class="info">
+                <h3 class="text">
+                    Multi-Post Stories Gain+Glory
+                </h3>
+                <ul class="languages">
+                    <li>HTML</li>
+                    <li>CSS</li>
+                    <li>Javascript</li>
+                </ul>
+                <a class="button" href="">See project</a>
+            </section>
+        </section>
+
+        <section class="card">
+            <section class="screenshot">
+                <img src="/Images/placeholder.png" alt="screenshot">
+            </section>
+            
+            <section class="info">
+                <h3 class="text">
+                    Multi-Post Stories Gain+Glory
+                </h3>
+                <ul class="languages">
+                    <li>HTML</li>
+                    <li>CSS</li>
+                    <li>Javascript</li>
+                    <li>Ruby on rails</li>
+                </ul>
+                <a class="button" href="">See project</a>
+            </section>
+        </section>
+
+        <section class="card">
+            <section class="screenshot">
+                <img src="/Images/placeholder.png" alt="screenshot">
+            </section>
+            
+            <section class="info">
+                <h3 class="text">
+                    Multi-Post Stories Gain+Glory
+                </h3>
+                <ul class="languages">
+                    <li>HTML</li>
+                    <li>CSS</li>
+                    <li>Ruby on rails</li>
+                </ul>
+                <a class="button" href="">See project</a>
+            </section>
+        </section>
+
     </section>
+
+
+    <section class="about">
+        <section class="aboutme">
+            <h1>About me</h1>
+            <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don't hestiate to contact me.</p>
+             <a class="button" href="">Get My Resume</a>
+        </section>
+
+        <section class="card">
+            <img src="/Images/languages-icon.svg" alt="languages">
+            <h3>Languages</h3>
+            <ul class="tags">
+                <li>Javascript</li>
+                <li>Ruby</li>
+                <li>HTML</li>
+                <li>CSS</li>
+            </ul>
+        </section>
```

Comment:
- [ ] [OPTIONAL] Kindly fix the image link as it looks broken
      your design 👇 
      
![image](https://user-images.githubusercontent.com/66097371/148616272-01d62319-b710-41ea-8884-d1526b947413.png)
    Figma design 👇 
       
![image](https://user-images.githubusercontent.com/66097371/148616329-c7eb07cb-15cc-4bd0-86b1-afa12a53251d.png)

     Kindly fix for all your card classes

Reply:
I don't see them broken no matter what I do, any clues on why or how to solve that?

Reply:
-  Try adding a correct path maybe you can name the path like this `src='./Images/Languages-icon.svg'` also try to get a live link of your project by hosting with GitHub so you can know how your work looks when live